.travis.yml

```diff
@@ -1,11 +1,22 @@
+env:
+  global:
+    - SIMPLECOV_ENABLED=1
+    - CC_TEST_REPORTER_ID=b7cacaaf8cd892926f18b6a494de52ade74bfc7f68befcba99b5adaa3823b662
+    - GIT_COMMITTED_AT=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then git log -1 --pretty=format:%ct; else git log -1 --skip 1 --pretty=format:%ct; fi)
```

Comment:
We can probably add `- JRUBY_OPTS=--dev` here to speed up spec execution on JRuby builds :)